styleguide/derek/view-templates/search/search.ejs

```diff
@@ -0,0 +1,54 @@
+<!--Header Medium-->
+<div class="imacBanner">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="imacBanner-textContainer imacBanner-headlineMedium">
+        <h2 class="imacBanner-headline">Search Results</h2>
+        <span class="imacBanner-line left-top"></span>
+        <span class="imacBanner-line left-bottom"></span>
+        <span class="imacBanner-line right-bottom"></span>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container half-padding">
+  <!-- Lead pattern -->
+  <div class="lead-wrapper">
+    <div class="lead-headerWrapper text-left">
+      <h1 class="lead-header">Search Results for: Cloud</h1>
+    </div>
+    <div class="lead-bodyWrapper text-left">
+      <p>About 6,030 results (0.31 seconds)</p>
+    </div>
+  </div>
+
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+  <%- partial('_search-result') %>
+
+</div>
+
+<div class="container half-padding">
+  <div class="search-pagerWrapper col-xs-12 col-sm-8 center-block">
+    <ul class="search search-pager">
+      <li class="search-pagerItem"><a href="javascript:void(0);">1</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">2</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">3</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">4</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">5</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">6</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">7</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">8</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">9</a></li>
+      <li class="search-pagerItem"><a href="javascript:void(0);">10</a></li>
+    </ul>
```

Comment:
The pager is what I pulled from the current dev branch for the solr build.
http://rsweb-9347-solr.www.dev.website.rackspace.com/

If there is an alternate version, I think it should happen there so that I at least would know the capabilities.